client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+const Suggestions = require( './suggestions-mixin' );
```

Comment:
@lamosty All `require` statements should now be replaced by `import`. Thx.

@gwwar I got rid of the mixin in [this commit](https://github.com/Automattic/wp-calypso/pull/8028/commits/e318fbe0da5abb8d514e3d9899c058c5dab2cc00). Any feedback is appreciated. Cheers.